index.html

```diff
@@ -1330,8 +1330,10 @@ <h2>DID Subject</h2>
 such as when a <a>DID resolver</a> is performing <a>DID resolution</a>.
 However, the fully resolved <a>DID document</a> always contains a valid
 <code><a>id</a></code> property. The value of <code><a>id</a></code> in the
-resolved <a>DID document</a> is expected to match the <a>DID</a> that was
-resolved.
+resolved <a>DID document</a> SHOULD match the <a>DID</a> that was
```

Comment:
Wait this isn't a MUST? Seems odd to me that I'd map did:example:123 => { "id": "did:example:456", ... }

Can we provide some further detail on why this is best left as a should?